includes/scheduler/class-actor.php

```diff
@@ -161,4 +167,44 @@ public static function sticky_post_update( $post_id ) {
 
 		self::schedule_profile_update( $post->post_author );
 	}
+
+	/**
+	 * Schedule a Delete activity when a user is deleted.
+	 *
+	 * @param int $user_id The user ID being deleted.
+	 */
+	public static function schedule_user_delete( $user_id ) {
+		// Don't bother if the user can't publish ActivityPub content.
+		if ( ! \user_can( $user_id, 'activitypub' ) ) {
+			return;
+		}
+
+		// Get the actor before deletion to ensure we have the data.
+		$actor = Actors::get_by_id( $user_id );
+
+		Tombstone::bury( $actor->get_id() );
+		Tombstone::bury( $actor->get_url() );
+
+		if ( \is_wp_error( $actor ) ) {
+			return;
+		}
```

Comment:
```suggestion
		$actor = Actors::get_by_id( $user_id );
		if ( \is_wp_error( $actor ) ) {
			return;
		}
		
		Tombstone::bury( $actor->get_id() );
		Tombstone::bury( $actor->get_url() );
```
Let's check for error before calling a method on it.